loops.js

```diff
@@ -32,7 +41,17 @@
 
 // Tip: een getal is oneven als je het door 2 deelt en er blijft nog iets over. Een getal is even als je het deelt door 2 en er blijft niets over..
 // ==========================================
+let evenofoneven;
```

Comment:
Wat je doet werkt, maar het is niet nodig om een let aan te maken. Als je in je loop kiest voor i < 16 en dan gelijk console.log in je if-statement doet, werkt het ook.